pkg/mkimage-raw-efi/efi-files/EFI/BOOT/grub.cfg

```diff
@@ -1,6 +1,6 @@
 gptprio.next -d dev -u uuid
 
 set root=$dev
-chainloader ($dev)/EFI/BOOT/BOOTX64.EFI
+chainloader ($dev)/PATH_TO_GRUB
```

Comment:
Can you please make more explicit that this is a variable that is going to be substituted?

The standard I've been trying to follow is to rename grub.cfg to grub.cfg.in and PATH_TO_GRUB to @PATH_TO_GRUB@